DNN Platform/DotNetNuke.Web.Deprecated/UI/WebControls/TermsSelector.cs

```diff
@@ -117,18 +115,10 @@ protected override void OnLoad(EventArgs e)
 				}
 			}
 
+			Page.ClientScript.RegisterClientScriptResource(GetType(), "DotNetNuke.Web.UI.WebControls.Resources.TermsSelector.js");
 
-			if (!Page.IsPostBack)
```

Comment:
I'll add this back, it was removed without my knowledge. Time gap between the start of this and finalizing the commit.